src/OIOProvider/index.js

```diff
@@ -16,15 +17,27 @@ export default class OIOProvider extends Component {
       className: PropTypes.string,
       fontFamily: PropTypes.string,
       fontSize: PropTypes.string,
-      style: PropTypes.object
+      fontWeights: PropTypes.object,
+      style: PropTypes.object,
+      textSizeScaleRatio: PropTypes.number,
+      textSizeMultiplier: PropTypes.number
    }
 
    static defaultProps = {
       children: null,
       className: '',
       fontFamily: 'sans-serif',
       fontSize: '16px',
-      style: {}
+      fontWeights: {
+         light: '300',
```

Comment:
Any reason these are strings and not numbers? Do units apply here? I can't remember how the responsive string things work with objects